packages/fuel-indexer-lib/src/graphql/base.graphql

```diff
@@ -38,7 +38,7 @@ enum IndexType {
     Hash,
 }
 
-directive @indexed(type: IndexType = BTree) on FIELD_DEFINITION | ENUM_VALUE
+directive @index(type: IndexType = BTree) on FIELD_DEFINITION | ENUM_VALUE
```

Comment:
It looks like we're changing the index directive to `@index` but the schema changes use `@indexed`. Was that intended?

Reply:
Not intended at all. Fix incoming